src/index.html

```diff
@@ -11,7 +11,145 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form method="post" action="https://mate-academy-form-lesson.herokuapp.com/create-application">
+      <fieldset class="fieldset-group">
+        <legend>Personal information</legend>
+        <label class="input-group">Surname:
+          <input
+            type="text"
+            id="surname"
+            name="surname"
+            autocomplete="off"
+          >
+        </label>
+        <label class="input-group" for="name">Name:
+          <input
+            type="text"
+            id="name"
+            name="name"
+          >
+        </label>
+        <label class="input-group" for="age">How old are You?
+          <input
+            type="number"
+            id="age"
+            name="age"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </label>
```

Comment:
No needs to use for/id when you wrap input into a label 
Apply all over the page
```suggestion
        <label class="input-group">How old are You?
          <input
            type="number"
            name="age"
            min="1"
            max="100"
            value="12"
          >
        </label>
```